### PR TITLE
Add workflow for updating the client based on API changes

### PR DESCRIPTION
**Pull request by josephschorr**

### CODE REVIEW

File: .github/workflows/manual-api-update.yaml [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+      - name: "Output update status"
+        env:
+          UPDATED_STATUS: "${{ steps.buf-update.outputs.updated }}"
+        run: |
+          echo "Update status: $UPDATED_STATUS"
+      - name: "Update package version"
+        uses: "authzed/actions/semver-update@main"
+        if: "steps.buf-update.outputs.updated == 'true'"
+        with:
+          sourcefile-path: "package.json"
```
Comment (samkim) [inline .github/workflows/manual-api-update.yaml:33]: pyproject.
```suggestion
          sourcefile-path: "pyproject.toml"
```

File: pyproject.toml [hunk 2]
```diff
@@ -2,13 +2,13 @@
 authors = ["Authzed <support@authzed.com>"]
 description = "Client library for SpiceDB."
 name = "authzed"
+version = "0.12.0"
```
Comment (samkim) [inline pyproject.toml:5]: Add a comment saying these 2 lines need to be together

File: .github/workflows/manual-api-update.yaml (hunk 1 above)
Comment (josephschorr) [reply .github/workflows/manual-api-update.yaml]: D'oh. Fixed

**Review by samkim — APPROVED** [review]
LGTM

**Pull request merged**